devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -204,8 +204,10 @@ def setLevel(value, rate = null) {
 	def additionalCmds = []
 	if (device.getDataValue("model") == "iQBR30" && value.toInteger() > 0) { // Handle iQ bulb not following spec
 		additionalCmds = zigbee.on()
-	} else if (device.getDataValue("manufacturer") == "MRVL") { // Handle marvel stack not reporting
+	} else if (isMRVL()) { // Handle marvel stack not reporting
 		additionalCmds = refresh()
+	} else if (isLeviton()) {
+		additionalCmds = zigbee.levelRefresh()
```

Comment:
@varzac would it work if we just treated this device as a zll device by moving the fingerprint over?

Reply:
Yes it should, but it would likely be a worse experience as on average it would be slower reporting of the level.  It would have the advantage of not requiring any local changes though.

Reply:
ok then, please let me know if I should revert some changes from current DTH.

Reply:
Yeah I'd revert these changes and move the relevant fingerprints to `zll-dimmer-bulb` with a correct `ocfDeviceType`

Reply:
I'm having second thoughts about it. 
Since Leviton DG3HL and Leviton DG6HD are not certified yet - moving those fingerprints may be an easy thing (from WWST certification point of view). 
But this issue occurs also in case of WWST certified device -> Leviton DL6HD (and maybe the other ones too). 
It should be reconsidered to leave those fingerprints here and fix this issue in a hub core.
@varzac @greens What do You think ?

Reply:
@KKlimczukS The migration isn't a big deal. It's just tedious, but I'm willing to take care of it.